cpp/include/cudf/copying.hpp

```diff
@@ -408,7 +408,8 @@ std::unique_ptr<column> shift(
  * @param indices A vector of indices used to take slices of `input`.
  * @return Vector of views of `input` indicated by the ranges in `indices`.
  */
-std::vector<column_view> slice(column_view const& input, std::vector<size_type> const& indices);
+std::vector<column_view> slice(column_view const& input, host_span<size_type const> indices);
+std::vector<column_view> slice(column_view const& input, std::initializer_list<size_type> indices);
```

Comment:
These should have their own doxygen. You should be able to use `@copydoc` of the span version and change the `@param indices` line to just say "Indices used to take slices of `input`"

Reply:
So this will render with 2 `@param indices`. One from the `@copydoc` and one included here.

I think it would be better to make the vector one more generic -- not use the word 'vector' and then you don't need the extra `@param indices` in the span version.

Reply:
Yup also saw the above. Added ingroup and changed wordings.

Reply:
I should mention that you can run `ninja docs_cudf` (or `make docs_cudf`) to build this locally and the `index.html` file will be created in `cpp/doxygen/html/` directory.